content/en/profiler/enabling/ssi.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Single Step Instrumentation for Profiling
+private: true
+---
+
+# Overview
+
+[Continuous Profiler][3] can be enabled as part of the [Single Step APM Instrumentation][1]
+(SSI) setup. Since this mode of profiler enablement is in private beta it is not yet integrated in
+the UI of the agent installation page. This page thus provides instructions on how to set it up. You
+should first familiarize yourself with SSI in general.
+
```

Comment:
```suggestion
(SSI) setup. Since this mode of profiler enablement is in private beta it is not yet integrated in
the UI of the agent installation page. This page provides instructions on how to set it up. You
should first familiarize yourself with SSI in general.

```